app/views/reviews/_navbar.html.erb

```diff
@@ -58,7 +58,7 @@
       <% end %>
 
       <p class="format_links">SOURCE FILES :
-        <a class="format_link" href="<%= record.umm_json_link %>" download>UMM-JSON</a> |
+        <a class="format_link" href="<%= record.umm_json_link %>" download target="_blank">UMM-JSON</a> |
         <a class="format_link" href="<%= record.native_link %>" download>NATIVE</a></p>
```

Comment:
should we open native as well in a new tab?

Reply:
I'm actually not entirely sure what the link is supposed to do. Currently it opens a dialog box that asks me to name a .native file for download. Is that the expected outcome?

Reply:
I guess so, maybe just keep it the way it is.   The browser probably doesn't know how to handle .native so it is causing a download.